app/src/main/java/org/oppia/android/app/customview/LessonThumbnailImageView.kt

```diff
@@ -23,7 +24,7 @@ class LessonThumbnailImageView @JvmOverloads constructor(
   context: Context,
   attrs: AttributeSet? = null,
   defStyleAttr: Int = 0
-) : AppCompatImageView(context, attrs, defStyleAttr) {
+) : ShapeableImageView(context, attrs, defStyleAttr) {
```

Comment:
This itself uses `AppCompatImageView`, under the hood and this is provided by `com.google.android.material.imageview`, so no extra third party requirements. 

And wit this we can make any of our image in any required shape by simply using any custom style like this `app:shapeAppearanceOverlay="@style/roundedCorners"`

Also, this doesn't affect any current parts of the app.

If you don't like this approach, the other thing I have is to wrap the LessonThumbnailImageView in XML under a card view to provide rounded corners.

Reply:
as for now, I had simply created a style of 4dp corners, but if we finalize to go with this solution, we can have a customized style independent of what corner radius we pass.

Reply:
I think we can use this.